test/integration/create_prediction_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class CreatePredictionTest < ActionDispatch::IntegrationTest
+  setup do
+    @user = User.create!(valid_user_params)
+  end
+
+  test 'create valid prediction' do
+    post '/api/predictions' + valid_query_string, valid_prediction_params
+    assert_equal 200, response.status
+    assert_equal Mime::JSON, response.content_type
+  end
+
+  test 'refuse to create invalid prediction' do
+    post '/api/predictions' + valid_query_string, invalid_prediction_params
+    assert_equal 422, response.status
```

Comment:
I think it would be worth testing the basic structure of the response. For example I think there should be an errors key with the errors.